### DISCUSSION TITLE
Do we need to rebuild if we are Setting up new environment ?

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-RC/community#308] [Issues]
Hello folks,
hope we are doing well!

I was working on search api and I wanted to get all records in one api call without limit.
for this I have added **search_limit=10000** in environment of rg service

is this a correct way? also Do i need to rebuild registry to see the change in api behaviour

@tejash-jl

**Comment by tejash-jl** [comment]
@paritshivani 
yes, the env is correct https://github.com/Sunbird-RC/community/blob/main/developer-documentation/configuration.md#search_limit
you need to force-recreate the docker container to reflect the new env values

**Comment by parthlawate** [comment]
@paritshivani long term you might want to use pagination in the longer term solution.
